store/rootmulti/utils.go

```diff
@@ -0,0 +1,144 @@
+package rootmulti
```

Comment:
I have a hunch these should live under the snapshot package, thoughts?

Reply:
See comment above. These utility methods are used for snapshot contents, not the metadata, so they belong with whatever does the actual data store snapshotting.